.circleci/config.yml

```diff
@@ -80,6 +81,10 @@ jobs:
       - heroku-buildpacks/install-build-dependencies
       - heroku-buildpacks/install-rust
       - heroku-buildpacks/install-libcnb-cargo
+      - run:
+          name: "Trust heroku/builder:22"
+          command: |
+            pack config trusted-builders add heroku/builder:22
```

Comment:
libcnb-test uses `--trust-builder` (as of https://github.com/heroku/libcnb.rs/pull/409), so this can be removed once the remaining cutlass tests are converted

Reply:
I briefly looked into running these with libcnb-test. I didn't pull the trigger mostly because cutlass has a (albeit simple) mechanism to compile meta-buildpacks and their dependent buildpacks. I don't think libcnb has functionality for this yet?

Reply:
Yeah libcnb-test doesn't have that yet.

(To clarify, I wasn't meaning to convert them now, just more of a potential cleanup once the conversion had occurred in the future)